config/database.yml.example

```diff
@@ -1,8 +1,37 @@
+#########################################################################
 # SQLite version 3.x
 #   gem install sqlite3
 #
 #   Ensure the SQLite 3 gem is defined in your Gemfile
 #   gem 'sqlite3'
+#   The configuration for SQLite has been done already.
+##########################################################################
+# PostgreSQL Configuration:
```

Comment:
Move this under the sqlite config.

Reply:
The configuration for the sqlite is already mentioned in the file please have a look at the code from line 35 till end. When user copies this file in the same folder and renamed it as 'database.yml' the configuration sets up itself.

Reply:
Yes, I know :)
Why are you mentioning it?

I asked you to leave comments of sqlite above sqlite config code, and move everything about pg (comments + configuration) below the sqlite things. 

And then what is missing is the mysql config.

Reply:
Okay now its clear